_includes/default_roles/facilitator.md

```diff
@@ -9,8 +9,9 @@ The _Facilitator_ conducts the meeting so that all of the agenda items are finis
 ***During the meeting***
 - [ ]  Facilitate the meeting according to following Tips below.
 
-{% if site.links.attendance != nil %}
 ***After the meeting***
+- [ ]  Update the [calendar invite][isc calendar] to include any newcomers.
```

Comment:
Should the scheduler take on this work they are updating the calendar already?

Reply:
Good idea

Reply:
I'm having trouble with this because it requires teaching the scheduler where to find who attended last time and where the attendance roster is kept and who is new.  Facilitator will already know who is new because they have to add in a new line for them.

Reply:
Seems easier to teach the Facilitator where to update the calendar rather than the other way around.

Reply:
> Seems easier to teach the Facilitator where to update the calendar rather than the other way around.

That's fair.

It's good for all the roles to know about the calendar.